source/_components/remote_rpi_gpio.markdown

```diff
@@ -113,3 +114,7 @@ switch:
     ports:
       17: Speaker Relay
 ```
+
+### Troubleshooting
+
+If you receive an error such as "gpiozero.exc.BadPinFactory: Unable to load any default pin factory!" try changing the pinfactory from "pigpio" to "mock", this adresses a known issue see: https://www.raspberrypi.org/forums/viewtopic.php?p=1417922
```

Comment:
Please use code fences instead of quotes (`"`). Instead of using a full link, try to link a piece of text.